packages/terra-theme-provider/src/ThemeProvider.jsx

```diff
@@ -32,11 +27,9 @@ const ThemeProvider = ({
     };
   }, [themeName]);
 
-  return children;
+  return <React.Fragment>{children}</React.Fragment>;
```

Comment:
why can't we directly pass back children?

Reply:
Short answer is no with our framework:

ERROR in ./packages/terra-theme-provider/src/ThemeProvider.jsx?dev-site-props-table
Module build failed (from ./node_modules/terra-dev-site/config/webpack/loaders/devSitePropsTable.js):
NonErrorEmittedError: (Emitted value instead of an instance of Error) Could not convert file to props table:
/Users/rm012685/releases/terra-framework/packages/terra-theme-provider/src/ThemeProvider.jsx
Error: No suitable component definition found.